### DISCUSSION TITLE
no puedo cambiar propiedad de "no requerido" a "requerido" en un campo de un formulario, llevo todo el dia sin poder hacerlo, recibo mensaje "bad request" "refresh"

### DISCUSSION THREAD

**Discussion by z0rro2k** [Cotalker/documentation#419] [4. Bug Reports] [answered]
![error pantalla no requerido a requerido](https://user-images.githubusercontent.com/39965925/133704968-7c1c1b09-26d0-4bcc-ae59-f55f8a0ed67a.png)
hola 

  a ver si alguien me da una mano:

  no puedo cambiar propiedad de "no requerido" a "requerido" en un campo de un formulario, llevo todo el dia sin poder hacerlo, recibo mensaje "bad request" "refresh"

gracias!

**Comment by NicolasPLG** [comment, marked as answer]
Hola @z0rro2k puedes revisar que otra parte del formulario tienes algo pendiente?. Lo revisé en otra compañía (ambiente prod) y no he tenido problemas. Quizás el error sucede cuando le pones "requerido" el mínimo sea 0. Debes colocar a lo menos 1 y luego requerido y allí no deberías tener problemas.

Te dejo un ejemplo cuando le pones requerido en una componente y con mínimo "0"

![image](https://user-images.githubusercontent.com/61419471/134002735-b054a12d-60c5-44a4-881c-9d233b71b65d.png)

Quedamos atentos ;)

**Comment by z0rro2k** [comment]
Hola

he hecho el cambio como sugeriste, y ahora si pude grabar el cambio y comprobar que el campo quedó como requerido.

muchas gracias por tu ayuda

PD. como cierro esta  pregunta o bug

**Reply by NicolasPLG** [reply]
Cierra como pregunta, ya que no es un bug la consulta que realizaste.

Atte.

**Comment by agallardol** [comment]
Marqué como respuesta lo que compartió @NicolasPLG. Muchas gracias!
